### DISCUSSION TITLE
Discuss key comparison somewhere?

### DISCUSSION THREAD

**Discussion by hoclun-rigsep** [re-rxjs/react-rxjs#231] [General]
I was thinking that there should be some mention of key equality in the relevant API docs, warning that key values should be primitives because `someMap.set([1,2],3).has([1,2])` is falsy, and then I started thinking, what if you did want to key on multiple properties? Should we give canonical advice how best to handle that?

**Comment by josepot** [comment]
> Should we give canonical advice how best to handle that?

Most definitely. In fact, we've had i[ssues opened in the past](https://github.com/re-rxjs/react-rxjs/issues/125) related to this topic. So, yes, most definitely, we should explain this better.
